index.html

```diff
@@ -582,9 +582,10 @@ <h3>Processing model</h3>
    <dd>"<code>no-cache</code>"
   </dl>
 
- <li><p>Let <var>response</var>’s <a>body</a> be a JSON <a>Object</a>
-  with a key "<code>value</code>"
-  set to the <a>JSON serialization</a> of <var>data</var>.
+ <li><p>Let <var>response</var>’s <a>body</a> be the UTF-8
+  encoded <a>JSON serialization</a> of a JSON <a>Object</a> with a key
+  "<code>value</code>" set to <var>data</var>. It is valid for the
+  constructed JSON <a>Object</a> to have more keys than this.
```

Comment:
It goes without saying that the object can be returned with more fields than `value`, precisely because there’s no wording to the effect of preventing them. I think if we _need_ to say anything about this, it should go in a note.

Further, arbitrary fields are allowed on nested objects as well, or at least that’s how driver implementations work in practice.

Reply:
I can drop that bit. The people I was talking to were interpreting the spec with the expectation that only the fields mentioned would ever be present on the returned payload, but I agree it should go without saying.

Reply:
Yes, but then it has come up on multiple occasions, so I think pointing it out somehow would be a great addition. If you want to submit another patch for that I will happily accept it.